test/e2e/framework/client.go

```diff
@@ -47,8 +47,9 @@ func NewPrometheusClientFromRoute(
 	routeClient routev1.RouteV1Interface,
 	namespace, name string,
 	token string,
+	ctx context.Context,
```

Comment:
nit: let's be consistent and put the context parameter in second place after the client